API.md

```diff
@@ -270,7 +270,7 @@ iD.fileFetcher.cache().imagery = [{
 
 Each imagery source should have the following properties:
 * `id` - Unique identifier for this source (also used as a url parameter)
-* `name` - Display name for the source
+* `name` - Display the name of the source
```

Comment:
This one was actually meant to read "display name", in the sense of a name to be displayed for the particular source.

```suggestion
* `name` - Label of the source
```